pretrain/tinyllama.py

```diff
@@ -150,11 +148,19 @@ def train(fabric, state, train_dataloader, val_dataloader, resume):
         fabric.print(f"Measured TFLOPs: {measured_flops * fabric.world_size / 1e12:.2f}")
         del meta_model, x
 
+    max_tokens_per_device = max_tokens // fabric.world_size
+    tokens_per_iter = micro_batch_size * model.config.block_size
```

Comment:
If the max seq length had been touched by an user, this should be

```suggestion
    tokens_per_iter = micro_batch_size * model.max_seq_length
```

I suggest doing it this way to minimize changes by users who play with this script

Reply:
Do you feel strongly about this? I prefer not to, because block size refers to the data here. Of course, for pretraining the two things would have the same value generally, but I still think it is good to differentiate for clarity across the board.